inc/migration.class.php

```diff
@@ -57,7 +57,11 @@ public static function getSQLFields(string $field_name, string $field_type): arr
                 if ($field_type === 'dropdown') {
                     $field_name = getForeignKeyFieldForItemType(PluginFieldsDropdown::getClassname($field_name));
                 }
-                $fields[$field_name] = "INT {$default_key_sign} NOT NULL DEFAULT 0";
+                if ($multiple == 1) {
+                    $fields[$field_name] = "LONGTEXT DEFAULT NULL";
```

Comment:
`TEXT` fields cannot have default value on MySQL.

Reply:
Yes sorry i change this